### PR TITLE
Update content app (remove Program.cs or Startup.cs examples)

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update content app (remove Program.cs or Startup.cs examples). I only removed the `Program.cs`/`Startup.cs` example and did not add any new explanation because there is already an example with an `IComposer` in this article.

Belongs to: 
https://github.com/umbraco/UmbracoDocs/issues/5709 

## Type of suggestion
* [x] Updated 'wrong' content

## Product & version (if relevant)
Umbraco 10, 11, 12 & 13

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @erikjanwestendorp thank you for the PR! 

We need to discuss about this PR with the CMS developers and when I get an answer on the conclusion I will get back to you here 💪

File: Reference/Notifications/SendingAllowedChildren-Notification/index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+versionFrom: 9.0.0
+versionTo: 10.0.0
+---
```
Comment (alina-tincas) [inline Reference/Notifications/SendingAllowedChildren-Notification/index.md:4]: Is there a reason why this was added? 🙈 These versions to and from where only used on our old legacy documentation so it is not needed at all on these current documentations. This file is also part of our old documentation repository 😓

**Comment by alina-tincas** [comment]
Hi @erikjanwestendorp I was wondering if you have seen my latest message and if perhaps it would be best to create a new PR?

**Comment by erikjanwestendorp** [comment]
@alina-tincas Sorry for the delay in this one! 😅  Something went wrong here and I think it is better to create a new PR indeed. 👍

**Pull request closed without merging**